pennylane/qchem/__init__.py

```diff
@@ -55,7 +55,7 @@
     core_matrix,
 )
 from .molecule import Molecule
-from .observable_hf import fermionic_observable, qubit_observable, jordan_wigner, simplify
+from .observable_hf import fermionic_observable, qubit_observable, jordan_wigner

```

Comment:
@soranjh, Just a heads up once this PR gets merged `qchem.simplify` will have been moved to `pauli.simplify`. Just so that you are aware. Anyways this will be removed all together in the upcoming PRs.

Reply:
Thanks @Jaybsoni. For the record, the following functions are moved from the `qchem` module to the `pauli` module:

```
simplify
_pauli_mult
_binary_matrix
```

Is that correct?

Reply:
the `_binary_matrix ` method from `tapering.py` was also moved into `pauli/utils.py`